app/services/webhook_service.py

```diff
@@ -86,6 +87,7 @@ async def process_webhook(webhook_request_data: WebhookRequestData):
         WebhookEvent.CONTENTS_FETCH_SUCCESS, WebhookEvent.CONTENTS_FETCH_FAILURE
     ] and Product.PUBLIC_CONTENT_SEARCH in settings.SUPPORTED_PRODUCTS:
         await process_contents_fetch_event(webhook_request_data=webhook_request_data)
+        await process_professional_contents_fetch_event(webhook_request_data=webhook_request_data)
```

Comment:
Depending on WP ID, we need to decide which one of the two to trigger
Have a whitelist of WP IDs for professional work platforms, if the WP ID belongs to that then process professional, else social

Ex: PROFESSIONAL_WORK_PLATFORM_IDS = ['<ID_FOR_LI>']